test/functional/fixtures/api/click/test_config.json

```diff
@@ -0,0 +1,3 @@
+{

```

Comment:
`test_config.json` belong to testcafe tests. They should be in `testcafe-fixtures` dir